gobblin-temporal/src/main/java/org/apache/gobblin/temporal/ddm/workflow/impl/ProcessWorkUnitsWorkflowImpl.java

```diff
@@ -61,4 +70,11 @@ protected NestingExecWorkflow<WorkUnitClaimCheck> createProcessingWorkflow(FileS
     // TODO: to incorporate multiple different concrete `NestingExecWorkflow` sub-workflows in the same super-workflow... shall we use queues?!?!?
     return Workflow.newChildWorkflowStub(NestingExecWorkflow.class, childOpts);
   }
+
+  protected CommitStepWorkflow createCommitStepWorkflow() {
+    ChildWorkflowOptions childOpts =
+        ChildWorkflowOptions.newBuilder().setParentClosePolicy(ParentClosePolicy.PARENT_CLOSE_POLICY_ABANDON).build();
```

Comment:
suggest to `.setWorkflowId` with something like `Help.qualifyNamePerExec`, as in `createProcessingWorkflow`